Jenkinsfile.buildchain

```diff
@@ -7,6 +7,8 @@ additionalExcludedArtifacts = "${env.ADDITIONAL_EXCLUDED_ARTIFACTS?.trim() ?: ''
 checkstyleFile = env.CHECKSTYLE_FILE?.trim() ?: null
 findbugsFile = env.FINDBUGS_FILE?.trim() ?: null
 pr_type = env.PR_TYPE?.trim() ?: null
+build_jdk_tool = env.BUILD_JDK_TOOL?.trim() ?: 'kie-jdk8'
+build_maven_tool = env.BUILD_MAVEN_TOOL?.trim() ?: 'kie-maven-3.6.3'
```

Comment:
how is this in line of our maven upgrade to 3.8.1 ? fyi https://issues.redhat.com/browse/BXMSPROD-1339

Reply:
upgrading Jenkinsfiles it is something that has to be done as a part of the https://issues.redhat.com/browse/BXMSPROD-1339:

- https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/3e968a15ca77bb99d9a1538ba2c4d6f55063fd99/Jenkinsfile.nightly#L8
- https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/3e968a15ca77bb99d9a1538ba2c4d6f55063fd99/Jenkinsfile.revapi#L17
- https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/3e968a15ca77bb99d9a1538ba2c4d6f55063fd99/Jenkinsfile.buildchain#L70

@akoufoudakis